test/README.md

```diff
@@ -0,0 +1,8 @@
+## Available Ginkgo Focuses
+
+### [CANARY]
```

Comment:
A note on why do we separate it, and how it is used in testing infrastructure might guide the next maintainer.
Is the idea that
- SMOKE will be run always against any environment and change?
- CANARY will be run before the release or nightly?

Reply:
Here are my initial thoughts,

- The nightly tests will run all integration tests which could run for hours.
- Canary tests will be run quite frequently multiple times in a day so we won't be able to run all of our integration tests. So we run a selected set of tests of most important features that need to be tested. And probably cross dependencies like ALB creation as part of Service creation etc..
- Smoke tests can be utilized to fail a workflow early, for example PR Integration workflow can run Smoke tests as prerequisite before running the lengthy integration tests.

I don't have much information about how upstream does this, so open to feedback to be more aligned with upstream testing or to improve this PR. Also I think as you suggested we need more clear guidance on types of tests and what should run in each type of test.

Reply:
Hi @abhipth  - This information is sufficient. You could please add this in the readme itself.
When we realize how upstream or other projects do any different for their benefits, we can evaluate and change it then.

But the above schedule idea and separation looks good to me.